lightning/src/ln/channelmanager.rs

```diff
@@ -1203,6 +1204,36 @@ impl ChannelDetails {
 	}
 }
 
+/// We store payments that are pending resolution as `PendingOutboundPayment` in `ChannelManager`,
+/// and they can take on either one of the following states as defined in this enum.
+#[derive(Debug, PartialEq)]
+pub enum PendingPaymentStatus {
+	/// When a payment is still being sent and awaiting successful delivery.
```

Comment:
I copied the docs for the enum cases here basically verbatim from [`PendingOutboundPayment`](https://github.com/lightningdevkit/rust-lightning/blob/main/lightning/src/ln/channelmanager.rs#L473). Let me know if there is a better/preferred way of handling this.